express-get/server.ts

```diff
@@ -0,0 +1,68 @@
+import express from 'express';
+import pg from 'pg';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.get('/api/films', async (req, res, next) => {
+  try {
+    const sql = `
+    select
+      "filmId",
+      "title",
+      "description",
+      "replacementCost"
+      from "films"
+      order by "replacementCost" desc
+      limit 2;
+    `;
+    const resp = await db.query(sql);
+    const films = resp.rows;
+    if (!films) {
```

Comment:
This will _always_ be truthy because `resp.rows` is an array, and arrays are always truthy.

Usually for get requests of entire collections, we just return an empty array if there aren't any rows. No need to check and throw a 404.